examples/index.html

```diff
@@ -89,8 +90,8 @@
         Minified version size: ~4kb
     </p>
     <br>
-    <a class="remodal-cancel" href="#">Cancel</a>
-    <a class="remodal-confirm" href="#">OK</a>
+    <span data-remodal-action="cancel" class="remodal-cancel">Cancel</span>
```

Comment:
Not questioning that, just suggesting that you keep it attached to a block-level element in this example (`<a>` or `<button>` maybe) rather than an inline element (`<span>`). Not a biggie though :+1: